sections/production/lockdependencies.polish.md

```diff
@@ -0,0 +1,69 @@
+# Zablokuj zależności
+
+<br/><br/>
+
+### Wyjaśnienie jednego akapitu
+
+Twój kod zależy od wielu zewnętrznych pakietów, powiedzmy, że „wymaga” i użyj momentjs-2.1.4, a następnie domyślnie po wdrożeniu do produkcji npm może pobrać momentjs 2.1.5, co niestety wprowadza kilka nowych błędów do tabeli. Użycie plików konfiguracyjnych npm i argumentu ``–save-exact = true`` instruuje npm, aby odwoływał się do *dokładnie* tej samej wersji, która została zainstalowana, więc następnym razem uruchomisz ``npm install`` (w wersji produkcyjnej lub w kontenerze Docker, który planujesz wysłać do testowania), zostanie pobrana ta sama zależna wersja. Alternatywnym i popularnym podejściem jest użycie pliku `.shrinkwrap` (łatwego do wygenerowania przy użyciu npm), który dokładnie określa, które pakiety i wersje powinny zostać zainstalowane, aby żadne środowisko nie mogło ulec pokusie pobierania nowszych wersji niż oczekiwano.
```

Comment:
"i użyj momentjs-2.1.4" should be: "i używa momentjs-2.1.4,"
"więc następnym razem uruchomisz" should be: "więc gdy następnym razem uruchomisz"

Reply:
Agree, fixed. Done, thanks a lot!